src/main/java/com/monumental/services/TagService.java

```diff
@@ -35,9 +26,13 @@ public TagService() {
     /**
      * Get all of the Tags with the specified name
      * @param name - name of the Tag to get as a String
+     * @param initializeLazyLoadedCollections - If true, initializes all of the Monuments associated with the Tags
      * @return List<Tag> - List of Tags with the specified name
      */
-    public List<Tag> getTagsByName(String name) {
-        return this.getFromWhere("name", name);
+    public List<Tag> getTagsByName(String name, boolean initializeLazyLoadedCollections) {
+        ArrayList<Criterion> criteria = new ArrayList<>();
+        criteria.add(Restrictions.eq("name", name));
+
+        return this.getWithCriteria(criteria, initializeLazyLoadedCollections);
```

Comment:
This is really smoove. There might be some other `get`s out there we can replace with this but I don't remember